packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/SystemServicesHostApiImpl.java

```diff
@@ -103,4 +104,17 @@ public String getTempFilePath(@NonNull String prefix, @NonNull String suffix) {
           null);
     }
   }
+
+  /**
+   * Returns whether or not a {@code SurfaceTexture} backs the {@code Surface} provided to CameraX
+   * to build the camera preview.
+   *
+   * <p>This is determined by the engine, who uses {@code SurfaceTexture}s on Android SDKs 29 and
+   * below.
+   */
+  @Override
+  @NonNull
+  public Boolean isUsingSurfaceTextureForPreview() {
```

Comment:
My 2c, and I'd love the thoughts by the other reviewers as well, as I could be wrong - I'd wish for this to say something about the property of the preview, instead of the technology. For example, `isPreviewTexturePreRotated` (or whatever best describes the situation).

If we agree, this change made elsewhere in the PR as well.

Reply:
Still open to making this change, just let me know!

Reply:
Renamed to `isPreviewPreTransformed`